src/invoices/InvoicesList/ExportSettingsModal/utils/createExportReport.js

```diff
@@ -151,6 +155,10 @@ function getInvoiceLineExportData({
     externalAccountNumber: getInvoiceLineExternalAccountNumbers(line, fundsMap, invalidReferenceLabel),
     referenceNumbers: getExportReferenceNumbers(line),
     lineTags: line.tags?.tagList?.join(' | '),
+    invoiceLineCreatedBy: line.metadata?.createdByUserId,
```

Comment:
Will only user IDs be included in the report? If so, is this expected behavior? (question applied for all `createdBy` and `updatedBy` in the PR)